README.md

```diff
@@ -1,2 +1,81 @@
-# repo-capston-project-01
-This is the first capston project.
+# Environment Action Conference & Expo 2022 Website
+
+> The objective of this capstone project is to build a suitable website to publicize the early arrival of a new edition of the conference and exhibition that Environment Action organizes every two years.
+
+![screenshot desktop](./screenshot-desktop.png)
+![screenshot mobile](./screenshot-mobile.png)
+
+> This conference focuses on keeping active and united the environmental initiatives that independent institutions and individuals in Latin America have been developing, seeking to rebuild the balance of the ecosystem in this part of the planet.
+
+
+## Features
+
+- The website has two pages working, home and about.
+- It displays the content in mobile and desktop viewport.
+
+### Mobile
+- It displays less information in some sections like speaker and foother.
+- It displays a hamburger menu button. By clicking on the icon it display the menu in a modal window.
+
+### Desktop
+- It displays more information that mobile.
+- It displays de menu at the top of the page.
+- It display user interactions like hover.
+
+## Built With
+
+- HTML / CSS / Javascript
+
+## Live Demo
+
+[Environmet Action](https://gabyse1.github.io/repo-capston-project-01/)
```

Comment:
- [ ] There is an issue with the deployment because this URL it's showing a 404 error. Please, kindly fix this deployment issue as it is required for this project :smiley: :raised_hands:
![image](https://user-images.githubusercontent.com/81522435/147198311-158764d6-20b1-4b8e-a14c-9da0f2b5d28c.png)